src/components/Product.jsx

```diff
@@ -0,0 +1,14 @@
+// eslint-disable-next-line react/prop-types
+function Product({label, imgSrc,altText,imgText, price}) {
+    return (
+        <article>
+            <span>{label}</span>
+            <img src={imgSrc} alt={altText}/>
+            <p>{imgText}</p>
+            <h4>{price}</h4>
```

Comment:
Hierbij mag het euroteken en ,- meegegeven worden in het component. Hierdoor hoeft alleen het cijfer zelf te worden doorgegeven.